roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -95,6 +95,20 @@
     - port: 443
       targetPort: "oaproxy"
 
+- name: Set memcached service
```

Comment:
my argument is just that I would like to avoid adding any new logic to `openshift_logging`, I'm alright with a new role if its a new, separate, stand alone component or with there being redundancy in only a few components if it will be deployed with them (and they will require it to be configured)